### PR TITLE
feat: Add sponsors section

### LINKED ISSUE #1651: Add sponsor section
**Describe the feature you'd like**
Add sponsor section

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anhanh11001**
Add sponsors section
LATER: I will make another PR to make a sponsor fragment to display the detail information of the sponsors. It is weird that when I fetched the sponsors, the number of sponsors sent to the app can go up to 600, I don't know if someone really made those much data or are there any problems in the server side. @iamareebjamal should I open an issue in the server

Fixes #1651

Screenshots for the change:

### CODE REVIEW

**Comment by angmas1** [comment]
A lot of sponsor's image is getting cut. Use different scale type or different shape of image view

**Comment by anhanh11001** [comment]
@angmas1 Updated

File: app/src/main/java/org/fossasia/openevent/general/sponsor/SponsorApi.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.fossasia.openevent.general.sponsor
+
+import io.reactivex.Single
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface SponsorApi {
+    @GET("events/{id}/sponsors?include=event&fields[event]=id&page[size]=0")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/sponsor/SponsorApi.kt]: Don't fetch all at once. Remove page size. In future, we can add paging

**Comment by angmas1** [comment]
> @angmas1 Updated

This looks good but, can you give some padding between image view and card view and make cards slightly larger and reduce space between adjacent cards and also reduce space between sponsor text views and cards.

**Comment by anhanh11001** [comment]
@angmas1 yes, I get your point! Doing now

**Comment by anhanh11001** [comment]
@angmas1 @iamareebjamal updated

**Pull request merged**